bodhi/tests/server/services/test_updates.py

```diff
@@ -1466,6 +1466,12 @@ def test_list_updates_rss(self):
         self.assertIn('application/rss+xml', res.headers['Content-Type'])
         self.assertIn('bodhi-2.0-1.fc17', res)
 
+    def test_list_updates_rss_with_filter(self):
+        res = self.app.get('/rss/updates/', {'severity': 'low'},
+                           headers={'Accept': 'application/atom+xml'})
+        self.assertIn('application/rss+xml', res.headers['Content-Type'])
+        self.assertIn('Filtered on: severity(low)', res)
+
```

Comment:
Maybe we should have a test where we do not filter, and one where we have more than one key